tools/buildtools/protein_structure_rebuilders/mypmfs/top_energies_to_matrix.xml

```diff
@@ -0,0 +1,54 @@
+<tool id = "mypmfs_matrix_generator" name = "MyPMFs Energy Matrix Generator" version="1.00">
+    <requirements>
+        <requirement type="package" version="0.1.8">mypmfs_py</requirement>
+    </requirements>
+    <command>
+        <![CDATA[
+        python '$__tool_directory__/top_energies_to_matrix.py' --protein_file '$protein_file' --txtmatrix '$txt_matrix' --csvmatrix '$csv_matrix' &&
+        cp ./data_potentials/top_energies.tsv '$tabular_data' &&
+        rm -rf pdb_files &&
+        rm -rf data_potentials &&
+        rm -rf training_dataset
+        ]]>
+        
+    </command>
+    <inputs>
+        <param name = "protein_file" type = "data" format = "txt" label = "Proteins" />
```

Comment:
```suggestion
        <param name="protein_file" type="data" format="txt" label="Proteins" />
```

If you like reformat your tool with the Galaxy-language-server extension for Visual Studio Code.